webapp/app/forms/steps/eligibility_steps.py

```diff
@@ -761,23 +771,20 @@ def __init__(self, endpoint, stored_data=None, *args, **kwargs):
         
     def _main_handle(self):  
         super()._main_handle()
-              
-        self.render_info.additional_info['dependent_notes'] = get_dependent_notes(self)
+        
+        dependent_notes = []
+    
+        if data_fits_data_model_from_list([UserAElsterAccountEligibilityData, UserBElsterAccountEligibilityData], self.stored_data):
+            dependent_notes.append(_l('form.eligibility.result-note.user_b_elster_account-registration-maybe'))
```

Comment:
I'm not sure this is correct? The text is for the case both persons have an elster account. `data_fits_data_model_from_list` is an OR condition. So if `UserAElsterAccountEligibilityData` returns true, `UserBElsterAccountEligibilityData` is never checked. So I think only having `UserBElsterAccountEligibilityData` here would be correct as it also checks that person_a_has_elster_account_eligibility is 'yes'.

Reply:
Oh thanks for the hint! :sweat_smile: